src/test/java/org/opensearch/security/dlic/rest/api/ActionGroupsApiTest.java

```diff
@@ -384,6 +385,8 @@ public void checkNullElementsInArray() throws Exception{
 
         String body = FileHelper.loadFile("restapi/actiongroup_null_array_element.json");
         HttpResponse response = rh.executePutRequest(ENDPOINT + "/CRUD_UT", body, new Header[0]);
+        Settings settings = Settings.builder().loadFromSource(response.getBody(), XContentType.JSON).build();
         Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusCode());
+        Assert.assertEquals(ErrorType.NULL_ARRAY_ELEMENT.getMessage(), settings.get("reason"));
```

Comment:
Use `AbstractConfigurationValidator.ErrorType.NULL_ARRAY_ELEMENT` to be consistent with the other references to `AbstractConfigurationValidator.ErrorType`.

Reply:
done